packages/schema/src/config/_common.ts

```diff
@@ -344,27 +344,23 @@ export default {
    *   // Will register redirect-ssl npm package
    *   'redirect-ssl',
    *   // Will register file from project server-middleware directory to handle /server-middleware/* requires
-   *   { path: '/server-middleware', handler: '~/server-middleware/index.js' },
+   *   { path: '/server-middleware/**', handler: '~/server-middleware/index.ts' },
```

Comment:
We can omit extension for being language agnostic:

```suggestion
   *   { path: '/server-middleware/**', handler: '~/server-middleware/my-middleware' },
```